bootcd/rpms/genesis_scripts/src/init.d-genesis

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+#
+# chkconfig: 2345 09 89
+# description: run genesis bootloader
+#
+
+### BEGIN INIT INFO
+# X-Start-After:    $network
```

Comment:
I think we should ask for more facilities to be up. I forget the upstart (ugh) parlance, but we want syslog, all interfaces, and boot settled before proceeding.